ethereum/eth/src/main/java/org/hyperledger/besu/ethereum/eth/sync/fastsync/worldstate/FastWorldStateDownloader.java

```diff
@@ -107,6 +107,10 @@ private IntSupplier downloadStateValue(final Function<FastWorldDownloadState, In
         return failed;
       }
 
+      Optional<BlockHeader> checkNull = Optional.ofNullable(fastSyncState.getPivotBlockHeader().get());
+      if(checkNull.isEmpty()){
+        LOG.error("Pivot Block not present");
```

Comment:
We would need to exit this method if no pivot block. either return Optional.empty or throw an exception

Reply:
Ok, thanks for suggestion